api/internal/account/serializers.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework import serializers
+from shared.django_apps.codecov_auth.models import (
+    Account,
+    InvoiceBilling,
+    StripeBilling,
+)
+
+from api.internal.owner.serializers import PlanSerializer
+
+
+class StripeBillingSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = StripeBilling
+        fields = ("customer_id", "subscription_id", "is_active")
+
+
+class InvoiceBillingSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = InvoiceBilling
+        fields = ("account_manager", "invoice_notes", "is_active")
+
+
+class BaseAccountSerializer(serializers.ModelSerializer):
+    stripe_billing = StripeBillingSerializer(many=True, read_only=True)
+    invoice_billing = InvoiceBillingSerializer(many=True, read_only=True)
+    plan = PlanSerializer(source="pretty_plan")
+
+    class Meta:
+        model = Account
```

Comment:
Curious if there's a reason why we want to expose this via this serializer (which ends up being REST) vs exposing it via graphql? 

We've had this "unspoken" desire to move off REST and do graphql if this is being surfaced to gazebo, so wondering if there was another use case for it

Reply:
Gotchaa, yeah those are valid points. I think the end goal here is to make resolvers for this (resolvers are mostly done by the application team) as we'll move off from rest, but I don't want to stop you from delivering this change. Since these aren't used in gazebo by now, I'd say part of the req for the applications to use this is to make resolvers as an active effort to transition off of rest - some plan related things are available in gql like the `plan` resolver, but I'm not expecting you to make the resolvers now and here (unless you're interested in learning and I can help you 🙌, it should be fairly easy to expose these via gql). 

If you merge this, could you make tickets in the apps team to consume these fields via gql?